test/groovy/BuildPluginStepTests.groovy

```diff
@@ -148,6 +148,28 @@ class BuildPluginStepTests extends BaseTest {
     assertJobStatusSuccess()
   }
 
+  @Test
+  void test_buildPlugin_with_forkCount() throws Exception {
+    def script = loadScript(scriptName)
+    // Run with forkCount=1C
+    script.call([forkCount: '1C'])
+    printCallStack()
+    // and confirm expected message is output
+    assertTrue(assertMethodCallContainsPattern('echo', 'Running parallel tests with forkCount'))
```

Comment:
Would it not be more relevant to assert the actual `-DforkCount` parameter in the command?

Reply:
I tried to assert the actual argument through multiple paths and could not find one that would detect it.  The tests do not display (or execute) the maven command, so the maven command is not visible in the output.  Since it is not visible in the output, I could not assert its presence or absence.

I used this technique (with a little bit of frustration) because it is the technique that is used by one of the other tests.  I think that the message may actually be helpful to users if they are puzzled if CI tests fail differently than their desktop tests.

Reply:
> The tests do not display (or execute) the maven command

Right but I thought there was something that mocked out the `sh` step so you can run assertions against it. But I am not very familiar with the tests in this repo.

Reply:
> the message may actually be helpful to users if they are puzzled if CI tests fail differently than their desktop tests

Perhaps, though if in doubt you should always refer to the actual command being run according to the log:

```
+ mvn …
```